<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Allocation</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>90</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="first_2">
       <property name="text">
        <string>first fit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="best_2">
       <property name="text">
        <string>best fit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="worst_2">
       <property name="text">
        <string>worst fit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>771</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="la">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>click on show to show memory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>insert</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>insert process size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Deallocate</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>179</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="add"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="siz"/>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>322</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>insert process ID and click  delete ex(1)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4"/>
   </widget>
   <widget class="QGraphicsView" name="toknow">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>splitter</zorder>
   <zorder>splitter_2</zorder>
   <zorder>Add</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>view</zorder>
   <zorder>la</zorder>
   <zorder>label_5</zorder>
   <zorder>insert</zorder>
   <zorder>label_8</zorder>
   <zorder>Show</zorder>
   <zorder>error</zorder>
   <zorder>Delete</zorder>
   <zorder>toknow</zorder>
   <zorder>first_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
